styles/styles.css

```diff
@@ -191,11 +191,10 @@ h5 {
 }
 
 h6 {
-  font-size: var(--heading-font-size-xxxs);
-  text-transform: uppercase;
-  margin-bottom: 1rem;
-  color: var(--secondary);
+  line-height: 1em;
```

Comment:
I think a lot of pages use the current styling for h6; here's two of mine for example

https://main--sunstar--hlxsites.hlx.live/about/business-performance
https://main--sunstar--hlxsites.hlx.live/about/history

I think all pages that use this will require re-authoring if they wanna keep the uppercase

Reply:
Thanks for pointing this @dnbute . Ideally they should be added to the visual tests for being flagged. I'd want to avoid reauthoring. Let me check.